.github/jitx-client/user.params

```diff
@@ -1,2 +1,6 @@
+server :
+  base-url = "https://app-testing.jitx.com"
+  url = "https://app-testing.jitx.com/api/"
+
```

Comment:
This points to the testing server, so it should be removed.

Reply:
I believe that it was added to allow the CI to run successfully. The normal CI fails on an error that is pending.

Reply:
I can remove it but the CI will fail when I do the push.

Reply:
And then the PR can't be merged :|